.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,32 @@
+name: Update dependencies
+
+on:
+  repository_dispatch:
+    types: [asyncapi/parser-js/release]
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Update dependency version
+        run: npm i ${{ github.event.client_payload.name }}@${{ github.event.client_payload.version }}
+
```

Comment:
we should run tests here, or make sure https://github.com/asyncapi/generator/blob/master/.github/workflows/pull-request-testing.yml is mandatory and if fails, automerge should not be performed....but actually we need both, as what is the point to create a PR anyway if tests fail

Reply:
Yeah, tests should pass. Is there an easy way to do this? I'm confused right now.

Reply:
just another step to run `npm test`, but I think we can survive with a PR that is created that later it cannot be merged because of failing tests, most important is to set in settings in branch protection requirement to make PR testing a mandatory check